### DISCUSSION TITLE
Name, Label, Description attributes

### DISCUSSION THREAD

**Discussion by DianeWold** [cdisc-org/DDF-RA#300] [Q&A]
Definitions for name, label, and description attributes (from USDJ_CT(3) generally follow the following patterns:
name: The literal identifier (i.e., distinction designation) of the <class>
label:  The short descriptive designation for the <class>
description: A narrative representation of the <class>

These definitions suggest to me that name and label, at least, are 1:1 with id.

Exceptions: 
Organization.name -- A non-unique textual identifier for the organization. (BRIDG)
SyntaxTemplate.name -- A narrative representation of the study eligibility criteria.
Timing.description -- The textual representation of the chronological relationship between temporal events.
EligibilityCriterion.description -- The literal identifier (i.e., distinctive designation) of the study eligibility criteria.

Why does the definition for EligibilityCriterion.description look like a definition for a name?

**Comment by BSnoeijerCD** [comment]
> Definitions for name, label, and description attributes (from USDJ_CT(3) generally follow the following patterns: name: The literal identifier (i.e., distinction designation) of the label: The short descriptive designation for the description: A narrative representation of the
> 
> These definitions suggest to me that name and label, at least, are 1:1 with id.
> 
> Exceptions: Organization.name -- A non-unique textual identifier for the organization. (BRIDG) SyntaxTemplate.name -- A narrative representation of the study eligibility criteria. Timing.description -- The textual representation of the chronological relationship between temporal events. EligibilityCriterion.description -- The literal identifier (i.e., distinctive designation) of the study eligibility criteria.
> 
> Why does the definition for EligibilityCriterion.description look like a definition for a name?

It seems like the definition for name and description are switched in the USDM CT spreadsheet. Can you create a public review JIRA ticket for it? Then we can handle it for the final version.

**Reply by DianeWold** [reply]
Created JIRA issue DDF-489 https://jira.cdisc.org/browse/DDF-489
In my original comment, I missed the fact that EligibilityCriterion.name was also an exception to the pattern for name attributes.
